arch/risc-v/src/common/espressif/Make.defs

```diff
@@ -120,8 +120,8 @@ ifeq ($(CONFIG_ESP_WIRELESS),y)
 	ifeq ($(CONFIG_ESPRESSIF_WIFI),y)
 		CHIP_CSRCS += esp_wifi_init.c
 		CHIP_CSRCS += esp_wlan.c
-		CHIP_CSRCS += esp_wifi_utils.c
 	endif
+	CHIP_CSRCS += esp_wifi_utils.c
```

Comment:
I cannot understand it, why we can enable wifi_utils, without init wifi?

Reply:
We are using that file just for one function instead of repeating code again which is called `esp_wifi_to_errno`. I will have a look again but there won't be a huge issue at least for now.